src/commons/api/history.rs

```diff
@@ -118,19 +58,20 @@ impl CommandHistory {
 
 impl fmt::Display for CommandHistory {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        writeln!(f, "time::command::key::success")?;
+        writeln!(f, "time::command::sequence::success")?;
 
         for command in self.commands() {
             let success_string = match &command.effect {
-                StoredEffect::Error { msg } => format!("ERROR -> {}", msg),
-                StoredEffect::Success { .. } => "OK".to_string(),
+                CommandHistoryResult::Init() => "INIT".to_string(),
+                CommandHistoryResult::Ok() => "OK".to_string(),
+                CommandHistoryResult::Error(msg) => format!("ERROR -> {}", msg),
             };
             writeln!(
                 f,
                 "{}::{} ::{}::{}",
```

Comment:
Given that above you say:
> `"time::command::sequence::success`

Is it correct that there is a space in `"{}::{} ::{}::{}"` (though this was also in the original code) ?